### PR TITLE
doc: Update the Quick start guide for ITK 5.0

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: docs/code/MixingITKAndNumPy.py [hunk 1]
```diff
@@ -14,34 +14,40 @@
 # Run filters on itk.Image
```
Comment (jhlegarreta) [inline docs/code/MixingITKAndNumPy.py]: In other comments class and method names have been surrounded by inverted comas. For the sake of consistency, I'd try to use that notation (or remove them from they were used), e.g. `itk.Image`.

File: docs/code/MixingITKAndNumPy.py [hunk 2]
```diff
@@ -14,34 +14,40 @@
...
 # Copy of itk.Image, data is copied
-np_copy = itk.GetArrayFromImage(itk_image)
+np_copy = itk.array_from_image(itk_image)
 
 
-# Do numpy stuff
+# Do numpy stuff...
 
 
 # Convert back to itk, view only, data is not copied
```
Comment (jhlegarreta) [inline docs/code/MixingITKAndNumPy.py]: `itk` has been capitalized in other comments. I would use the capitalized version everywhere.

File: docs/code/MixingITKAndNumPy.py [hunk 3]
```diff
@@ -14,34 +14,40 @@
...
+itk_np_view = itk.image_view_from_array(np_copy)
 
 # Convert back to itk, data is copied
-itk_np_copy = itk.GetImageFromArray(np_copy)
+itk_np_copy = itk.image_from_array(np_copy)
 
 # Save result
 itk.imwrite(itk_np_view, output_filename)
 
 # Vnl matrix from array
```
Comment (jhlegarreta) [inline docs/code/MixingITKAndNumPy.py]: The same comment applies to `Vnl` vs. `VNL`.

File: docs/code/MixingITKAndNumPy.py (hunk 1 above)
Comment (thewtex) [reply docs/code/MixingITKAndNumPy.py]: Great observation -- I have removed the backticks from the other locations.

File: docs/code/MixingITKAndNumPy.py (hunk 2 above)
Comment (thewtex) [reply docs/code/MixingITKAndNumPy.py]: Good point -- capitialized!

File: docs/code/MixingITKAndNumPy.py (hunk 3 above)
Comment (thewtex) [reply docs/code/MixingITKAndNumPy.py]: All `VNL` now!

**Pull request merged**